package.json

```diff
@@ -147,7 +149,7 @@
     "vite-plugin-istanbul": "5.0.0"
   },
   "resolutions": {
-    "@graasp/sdk": "2.0.1"
+    "@graasp/sdk": "3.2.0"
```

Comment:
If you cascaded this version in all other packages you don't need to resolve it.